integreat_cms/cms/templates/events/event_form_sidebar/venue_box.html

```diff
@@ -39,8 +39,9 @@
             </div>
         </div>
         <p class="text-sm italic block mt-2 mb-2">
-            {% translate "Create an event location or start typing the name of an existing location" %}.
+            {% translate "Create an event location or start typing the name of an existing location. Only published locations can be set as event venues" %}.
         </p>
+        <div id="poi-form-widget"></div>
```

Comment:
```suggestion
```
To me, it would make more sense to move this down instead of having it between the search field and the preview of the previous address.